### PR TITLE
display errors

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/FileProcessor.js [hunk 1]
```diff
@@ -113,8 +113,10 @@ class FileProcessor {
         }
       }
       this.data.push(resultObject);
-      //TODO: errors aren't bubbling up
-      //need to think about this since we don't want to throw a parse error.  or do we?
+      // display any parse errors but do not bubble them up:
+      if (err) {
+        console.log(`${err.name} in file ${file}: ${err.message}`);
```
Comment (jgallen23) [inline lib/FileProcessor.js]: we should bubble the error back up

Comment (jgallen23) [reply lib/FileProcessor.js]: make sure to add a test too (if none)
